src/db/collections/milestone.rs

```diff
@@ -143,22 +146,26 @@ impl MongoDb {
             .transpose()?)
     }
 
-    /// Gets the timestamp of a milestone by the [`MilestoneIndex`].
-    pub async fn get_milestone_timestamp(&self, index: MilestoneIndex) -> Result<Option<MilestoneTimestamp>, Error> {
```

Comment:
Why remove this?

Reply:
We never called it, so I think it's save to delete.